### DISCUSSION TITLE
Which parameters would you recommend changing to increase inference speed?

### DISCUSSION THREAD

**Discussion by JudeWells** [chaidiscovery/chai-lab#141] [Q&A] [answered]
I would like to quickly score thousands of sequence variations. Do you have any guidance on what these parameters can be reasonably reduced to?
num_trunk_recycles
num_diffn_timesteps
would 100 or 50 time-steps still give reasonable results? and are there any other settings that can be adjusted to increase inference speed?

**Comment by arogozhnikov** [comment, marked as answer]
Hi Jude,

number one parameter to tune is the number of gpus in your cluster :)

That factor aside, you can find some guidance in #80 on diffusion, and I'd recommend to keep num_trunk_recycles=3.
